content/docs/command-reference/plots/diff.md

```diff
@@ -105,20 +105,32 @@ file:///Users/dmitry/src/plots/logs.html
 Compare two specific versions (commit hashes, tags, or branches):
 
 ```dvc
-$ dvc plots diff HEAD 0135527 --targets logs.csv
+$ dvc plots diff --targets logs.csv -- HEAD 0135527
 file:///Users/usr/src/plots/logs.csv.html
 ```
 
 ![](/img/plots_diff.svg)
 
-> Alternatively, you can also run above statement as:
+> Please note that when you're specifying `--targets` options with multiple
+> `revisions`:
+>
+> ```dvc
+> $ dvc plots diff --targets t1.json t2.csv HEAD v1 v2
+> ```
+>
+> This statement will show an error because argument parser confuses `revisions`
+> as arguments for `--targets` option. To avoid this error use `--` after all
+> the arguments for `--targets`:
```

Comment:
Let's move this note up to the description as a regular paragraph, and just make a quick reminder in this note (block quote) here in the example.

Reply:
Actually, per https://github.com/iterative/dvc.org/pull/1544#discussion_r452406892 please move up into the `--targets` option bullet text instead. Feel free to use several paragraphs and even code blocks or block quote notes in there, but do try to summarize all this a little in both places. Thanks

Reply:
please, let's try to keep it really simple. The problem itself is very simple and clear and one sentence should be enough in the description or option description to my mind.

Reply:
Yeah, by "summarize all this a little" I meant summarize it a lot too 🙂

Reply:
Continued in https://github.com/iterative/dvc.org/pull/1549#pullrequestreview-446819554